com.microsoft.mrtk.input/Visualizers/Visuals/Models/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
We should make sure this is added to our cgmanifest

Reply:
Updated!